### PR TITLE
Renaming SmartSignalsManagementApi

### PR DESCRIPTION
**Pull request by efvink**
Renaming SmartSignalsManagementApi - namespace, project name, SmartSignal to SmartDetector, signalResults to Alerts

### CODE REVIEW

File: src/management/server/SmartDetectorsManagementApi/Models/AlertRuleApiEntity.cs [hunk 1]
```diff
@@ -27,25 +27,25 @@ public class AlertRuleApiEntity
         public string Description { get; set; }
 
         /// <summary>
-        /// Gets or sets the signal ID.
+        /// Gets or sets the Smart Detector ID.
         /// </summary>
-        [JsonProperty("signalId")]
-        public string SignalId { get; set; }
+        [JsonProperty("SmartDetectorId")]
```
Comment (EranGon) [inline src/management/server/SmartDetectorsManagementApi/Models/AlertRuleApiEntity.cs]: >S [](start = 23, length = 1)

lower case 's'

Comment (EranGon) [inline src/management/server/SmartDetectorsManagementApi/Models/AlertRuleApiEntity.cs]: >SmartDetectorId [](start = 23, length = 15)

doesn't this break the UX?

File: src/management/server/SmartDetectorsManagementApi/Responses/ListSmartDetectorsResponse.cs [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-    public class ListSmartSignalsResponse
+    public class ListSmartDetectorsResponse
     {
         /// <summary>
-        /// Gets or sets the smart signals list
+        /// Gets or sets the Smart Detectors list
         /// </summary>
-        [JsonProperty("signals")]
-        public IList<Signal> Signals { get; set; }
+        [JsonProperty("smartDetectors")]
```
Comment (EranGon) [inline src/management/server/SmartDetectorsManagementApi/Responses/ListSmartDetectorsResponse.cs]: >smartDetectors [](start = 23, length = 14)

doesn't this break the UX?

Comment (efvink) [reply src/management/server/SmartDetectorsManagementApi/Responses/ListSmartDetectorsResponse.cs]: Yair will take care of it 

---
In reply to: [175383416](https://github.com/liadtal/Smart-Alerts/pull/102#discussion_r175383416) [](ancestors = 175383416)

File: src/management/server/SmartDetectorsManagementApi/Models/AlertRuleApiEntity.cs (hunk 1 above)
Comment (efvink) [reply src/management/server/SmartDetectorsManagementApi/Models/AlertRuleApiEntity.cs]: Yair will take care of it 

---
In reply to: [175383385](https://github.com/liadtal/Smart-Alerts/pull/102#discussion_r175383385) [](ancestors = 175383385)

**Pull request merged**